controllers/admin/AdminInvoicesController.php

```diff
@@ -99,7 +99,7 @@ public function __construct()
                     ),
                     'PS_INVOICE_LEGAL_FREE_TEXT' => array(
                         'title' => $this->trans('Legal free text', array(), 'Admin.OrdersCustomers.Feature'),
-                        'desc' => $this->trans('Use this field to display additional text on your invoice, like specific legal information. It will appear below the payment methods summary.', array(), 'Admin.OrdersCustomers.Help'),
+                        'desc' => $this->trans('Use this field to show additional informations on the invoice,below the payment methods summary (like e.g. specific legal informations).', array(), 'Admin.OrdersCustomers.Help'),
```

Comment:
Thank you Rainer :)
I hadn't noticed but "information" should be singular too. Could you please change it too? We'll merge after this!